src/koreo_tooling/function_test.py

```diff
@@ -16,10 +15,7 @@
     UnwrappedOutcome,
     is_unwrapped_ok,
 )
-
-
-from koreo.function_test.run import run_function_test
-from koreo.function_test.structure import FunctionTest
+from lsprotocol import types
```

Comment:
Man. This screws my brain up so badly because I think of this as being "almost" a part of this tool... that is, more local than true 3p stuff.